### PR TITLE
clean up README.md, fix default username

### PR DESCRIPTION
**Pull request by notque**
Tried to clean up the README. Also found an error with the default username for es. You may not appreciate the choices, and that's fine. I was going over the code to test it out and it just didn't seem as clean as I'd go for.

### CODE REVIEW

File: cmd/palomar/README.md [hunk 1]
```diff
@@ -1,37 +1,33 @@
-# palomo
+# Palomar
 
-An elasticsearch/opensearch frontend and ATP repo crawler meant to provide
-search services for the bluesky network.
+Palomar is an Elasticsearch/OpenSearch frontend and ATP (ActivityPub) repository crawler designed to provide search services for the Bluesky network.
+
+## Prerequisites
+
+- GoLang (version 1.x)
```
Comment (whyrusleeping) [inline cmd/palomar/README.md:7]: 1.20 is where we're at now

File: cmd/palomar/README.md [hunk 2]
```diff
@@ -44,8 +40,7 @@ index up to date you will periodcally need to scrape the data.
 ## API
 
 ### `/index/:did`
-Index the content in the given users repo. Keeps track of the last repo update
-and only fetches incremental changes
+Indexes the content in the given user's repository. It keeps track of the last repository update and only fetches incremental changes.
 
 ### `/search?q=QUERY`
-Very simple case-insensitive search results from across the entire app.
+Performs a simple, case-insensitive search across the entire application.
```
Comment (whyrusleeping) [inline cmd/palomar/README.md:46]: these routes are actually different now, probably needs some updating

Comment (notque) [reply cmd/palomar/README.md:46]: I've tried to determine what these are, and I can't sort it out. Over time I'll have a better understanding of how everything interacts, but it isn't clear to me yet.

File: cmd/palomar/README.md [hunk 3]
```diff
@@ -1,37 +1,33 @@
-# palomo
+# Palomar
 
-An elasticsearch/opensearch frontend and ATP repo crawler meant to provide
-search services for the bluesky network.
+Palomar is an Elasticsearch/OpenSearch frontend and ATP (ActivityPub) repository crawler designed to provide search services for the Bluesky network.
```
Comment (bnewbold) [inline cmd/palomar/README.md:3]: looking at this now, we are prefering `atproto` over `ATP` in this kind of context. but more importantly, this isn't "ActivityPub", it is "AT Protocol".

File: cmd/palomar/README.md [hunk 4]
```diff
@@ -1,37 +1,33 @@
...
+
+## Prerequisites
+
+- GoLang (version 1.x)
+- Running instance of Elasticsearch or OpenSearch for indexing.
+- Valid credentials for the Personal Data Store (PDS) you want to index against.
```
Comment (bnewbold) [inline cmd/palomar/README.md:9]: you don't need PDS credentials. this service consumes from a full firehose feed, which doesn't not require authentication.

File: cmd/palomar/README.md [hunk 5]
```diff
@@ -1,37 +1,33 @@
...
 ```
 
-## Setup
-
-You will need a running elasticsearch instance (or cluster) for indexing, and
-valid credentials for the PDS you wish to index against.
+## Configuration
 
-The following environment variables should be set:
+Palomar uses environment variables for configuration. Here are the required ones:
```
Comment (bnewbold) [inline cmd/palomar/README.md:19]: these are not all "required", only some of them are necessary. the `--help` on the CLI clarifies, I think.

**Review by bnewbold — CHANGES_REQUESTED** [review]
a couple doc notes. the "ActivityPub" confusion in particular should not get merged as-is

**Comment by notque** [comment]
I've attempted to address the issues. Please let me know if you'd prefer something different.

File: cmd/palomar/README.md (hunk 5 above)
Comment (notque) [reply cmd/palomar/README.md]: I do not see a help option, but perhaps I'm missing it.

File: cmd/palomar/README.md (hunk 3 above)
Comment (notque) [reply cmd/palomar/README.md]: Understood, my mistake.

File: cmd/palomar/README.md (hunk 5 above)
Comment (bnewbold) [reply cmd/palomar/README.md]:
```
bnewbold@snocat:~/code/indigo$ go run ./cmd/palomar/ --help
NAME:
   palomar - search indexing and query service (using ES or OS)

USAGE:
   palomar [global options] command [command options] [arguments...]

VERSION:
   dev-

COMMANDS:
   elastic-check  
   search         run a simple query against search index
   run            combined indexing+query server
   help, h        Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --elastic-cert-file value  certificate file path [$ES_CERT_FILE, $ELASTIC_CERT_FILE]
   --elastic-username value   elasticsearch username (default: "admin") [$ES_USERNAME, $ELASTIC_USERNAME]
   --elastic-password value   elasticsearch password (default: "admin") [$ES_PASSWORD, $ELASTIC_PASSWORD]
   --elastic-hosts value      elasticsearch hosts (schema/host/port) (default: "http://localhost:9200") [$ES_HOSTS, $ELASTIC_HOSTS]
   --es-post-index value      ES index for 'post' documents (default: "posts") [$ES_POST_INDEX]
   --es-profile-index value   ES index for 'profile' documents (default: "profiles") [$ES_PROFILE_INDEX]
   --bgs-host value           hostname and port of BGS to subscribe to (default: "https://bsky.social") [$ATP_BGS_HOST]
   --plc-host value           method, hostname, and port of PLC registry (default: "https://plc.directory") [$ATP_PLC_HOST]
   --pds-host value           method, hostname, and port of PDS instance (default: "https://bsky.social") [$ATP_PDS_HOST]
   --help, -h                 show help
   --version, -v              print the version
```

**Review by bnewbold — APPROVED** [review]

**Pull request merged**